recipe/meta.yaml

```diff
@@ -39,6 +39,7 @@ requirements:
     - requests
     - urllib3 >=1.25
     - pytorch
+    - cryptography <40.0.2,>=3.3.2
```

Comment:
I don't like capping cryptography, that's an extremely low-level but highly security-critical package, with a very conservative evolution speed. My preference would be to patch the upper bound completely.

And if there's a good argument for not doing that, then at least please order this sensibly, i.e.
``` 
- cryptography >=3.3.2,<40.0.2
```

Reply:
The awscli has the cap, not torchdata directly. The awscli stuff doesn't do a `pip check` so this is not caught there.

Reply:
Should this `cryptography` pin be placed under test.requires instead of being listed as a runtime dependency actually?

Reply:
@weiji14 - I am not very familiar with torchdata. Feel free to take over this PR if you have any ideas how to push it forward.

Reply:
Ok, let me open another PR to add myself to the recipe maintainer list.